css/style.css

```diff
@@ -266,12 +269,18 @@ input[type="checkbox"] {
 /* ----------- UI - Graph ----------- */
 
 .vis {
-  display: grid;
```

Comment:
This also suffers from the same issue as I mentioned above. With Grid, all we can say is: take up X amount of height relative to other rows in the grid or the height of the page. What we really want is this `.vis` element's child elements (rows in this case) to only take up as much space as they need to given their content and padding, and we want the `#visualization` area to absorb the rest of the space.

Inspecting this grid in Nightly (with its cool Layout inspector!) shows us that the headers at the top are taking up more space than they need to be, taking away space from the visualization area.

![screen shot 2017-08-16 at 8 38 26 am](https://user-images.githubusercontent.com/17437436/29371688-e7d644e8-825d-11e7-8948-4a23439e3a42.png)